lib/widgets/app_scaffold.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:coffeecard/base/strings_environment.dart';
+import 'package:coffeecard/base/style/colors.dart';
+import 'package:coffeecard/base/style/text_styles.dart';
+import 'package:coffeecard/cubits/environment/environment_cubit.dart';
+import 'package:coffeecard/widgets/components/dialog.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:gap/gap.dart';
+
+class AppScaffold extends StatelessWidget {
+  final Widget? title;
+  final Widget body;
+  final bool resizeToAvoidBottomInset;
+
+  AppScaffold({
```

Comment:
I still want two constructors here, this is more unclear than what we had before

Reply:
I don't agree with having the scaffold be in either 'normal' or 'login' mode, this widget should not be that specific and not writing specific code that only a single page will use, i much prefer taking the general approach as i have done here and making the title nullable and determine the behaviour of the widget on wheter this is null or not. I agree that checking if the String title is null maybe be additional code, but  then i propose to remove the ```String? title``` field and adding ```this.title``` instead. Then we can simply check if the title widget is given or not and we can reuse this other places where a ttitle may not be required

Reply:
with that said i may have an idea, stay tuned

Reply:
i pushed a new constructor, can you review and resolve this if you agree?